infra/aws/terraform/prow-build-cluster/docs/Debug.md

```diff
@@ -1,28 +1,25 @@
-## SSH Access EKS Nodes
-
-To access cluster nodes via ssh, first you need to provision bastion host:
-
-```bash
-TF_ARGS="-var bastion_install=true" make apply
-```
-
-After successful apply, terraform outputs should get updated with ip address of the bastion host.
-Use following snippet for accessing cluster node:
-
-```bash
-# Access to private key is limited to cluster administrators.
-ssh-add <path_to_private_key>
-
-# Example node name copied from the result of `kubectl get nodes`.
-export TARGET_NODE='ip-10-1-64-11.us-east-2.compute.internal'
-
-export BASTION_IP=$(make output | jq '.bastion_ip_address.value' | tr -d '"')
-
-ssh -A -J ubuntu@${BASTION_IP} ubuntu@${TARGET_NODE}
-```
-
-**Remember to remove the bastion host after debugging.**
-
-```
-make apply
-```
+## Access the EKS cluster nodes
+
+> [!NOTE]
+> The bastion host and SSH keys have been removed in favor of [AWS Systems Manager (Fleet Manager)](https://docs.aws.amazon.com/systems-manager/latest/userguide/what-is-systems-manager.html).
+
+To access the EKS cluster nodes, you need to login to the appropriate AWS account, go to
```

Comment:
we can add this https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/connect-with-systems-manager-session-manager.html ?

Reply:
@ameukam I added a link to this document, but I left the original instructions as-is because I don't find this document to be that detailed.